pubspec.yaml

```diff
@@ -1,7 +1,7 @@
 name: facebook_app_events
 description: Flutter plugin for Facebook App Events, an app measurement
   solution that provides insight on app usage and user engagement in Facebook Analytics.
-version: 0.9.0
+version: 0.10.0
```

Comment:
Don't bump the version in the PR since it will be coordinated with if there are other PRs to include in the same release.